pkg/metrics/queue.go

```diff
@@ -99,15 +100,15 @@ func InitQueueMetrics(name string) *QueueMetrics {
 			Namespace:   Namespace,
 			Name:        "queue_resource",
 			ConstLabels: prometheus.Labels{"queue": name},
-			Help:        "Queue resource metrics. State of the resource includes `guaranteed`, `max`, `allocated`, `pending`, `preempting`.",
+			Help:        "Queue resource metrics. State of the resource includes `guaranteed`, `max`, `allocated`, `pending`, `preempting`, 'maxRunningApps'.",
```

Comment:
nit: please use backquotes (`) instead of single quotation marks (').

Reply:
Good catch, thanks!